src/main/java/com/blog/dao/impl/TagDaoRestClientImpl.java

```diff
@@ -46,4 +61,43 @@ public TagDaoRestClientImpl(RestTemplate restTemplate, HttpHeaders headers) {
         );
         return tags.getBody();
     }
-}
+
+    public void addTag(Tag tag) {
+        LOGGER.debug("Add new tag = [{}].", tag);
+
+        entity = new HttpEntity<>(convertToJson(tag), headers);
+
+        ResponseEntity<String> responseEntity = restTemplate.exchange(
+                createURLWithEndpoint(endpoint),
+                HttpMethod.POST,
+                entity,
+                String.class
+        );
+    }
+
+    public void updateTag(Tag tag) {
+        LOGGER.debug("Update tag = [{}].", tag);
+
+        entity = new HttpEntity<>(convertToJson(tag), headers);
+
+        ResponseEntity<String> responseEntity = restTemplate.exchange(
+                createURLWithEndpoint(endpoint),
+                HttpMethod.PUT,
+                entity,
+                String.class
+        );
+    }
+
+    public void deleteTag(Long tagId) {
```

Comment:
Where is exception handling? What if service unavailable? At least response status code should be validated